nexuse2e-webapp/src/main/frontend/src/app/conversation-status-counts/conversation-status-counts.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { DataService } from "../services/data.service";
+
+@Component({
+  selector: "app-conversation-status-counts",
+  templateUrl: "./conversation-status-counts.component.html",
+  styleUrls: ["./conversation-status-counts.component.scss"],
+})
+export class ConversationStatusCountsComponent implements OnInit {
+  @Input() dashboardTimeFrameInDays = 0;
+  conversationStatusCounts: { [status: string]: number } = {};
+  totalCount = 0;
+  objectKeys = Object.keys;
+  loaded = false;
+
+  constructor(private dataService: DataService) {}
+
+  async ngOnInit(): Promise<void> {
+    this.conversationStatusCounts = await this.dataService.getConversationStatusCounts();
+    this.getConversationStatusTotalCount();
+  }
+
+  getConversationStatusTotalCount() {
+    this.loaded = false;
```

Comment:
Why setting this here? It is always already false from the property init and if not we should set it before we make the call to the backend and not only while we make the in memory processing.

Reply:
You are right, done.